src/app/get-involved/donate-blood/TabSection/index.tsx

```diff
@@ -10,26 +11,52 @@ import {
     TabPanel,
     Tabs,
 } from '#components/Tabs';
+import WhereToDonateCard from '#components/WhereToDonateCard';
 
 import whereToDonate from './data';
 
 import styles from './styles.module.css';
 
 export default function TabSection() {
     const [value, setValue] = useState<string | undefined>('where');
+
+    const whereToDonateItems = whereToDonate.map((item) => ({
+        id: item.id,
+        title: item.title,
+        description: isDefined(item.description)
+            ? <WhereToDonateCard whereToDonateItems={item.description} />
+            : undefined,
+    }));
+
+    const DonatingPersonApplyHasTo = [
+        'Be 18 to 60 years old',
+        'Weight above 45 kg',
+        'Have hemoglobin above 12 gm/dl',
+        'Have blood pressure 110-160 / 70-96 mmHg',
+        'Not to be pregnant, breastfeeding, and have at least 8 days since the start of the recent menstruation',
+        'Not having recent use of drugs or strong medicines (people who take strong medicine for a short period will not able to donate blood from one week up to 2 years)',
+        'Not to had a medical surgery for 2 years',
+    ];
+
+    const donaterOrganisationApplyHasTo = [
+        'We are visiting daily in companies and organizations to collect blood.It is a great activity for team building and recreational days,and social responsibility events.',
+        'Arrange a regular or one-time blood donation at your company or organization by contacting Mr. Ram Bahadur Shrestha via email,ram.shrestha@nrcs.org, or phone, 9841398605.',
+        'The earlier we set the date, the easier we can plan the efficient blood supply management. But if needed, we can deploy our blood collection team with even short notice! Never think it is too late or too early – today is just the perfect time to start planning the blood donation event in your community or company!',
+    ];
+
     return (
         <Tabs
             value={value}
             onChange={setValue}
         >
             <TabList>
-                <Tab name="where">
+                <Tab name="where" className={styles.tab}>
                     Where to donate?
                 </Tab>
-                <Tab name="can">
+                <Tab name="can" className={styles.tab}>
                     Can i donate?
                 </Tab>
-                <Tab name="organisation">
+                <Tab name="organisation" className={styles.tab}>
                     Donate blood as an organisation
                 </Tab>
```

Comment:
The tab names should be more meaningful.
Eg: where-to-donate, can-i-donate, donate-blood